app/views/teacher_interface/application_forms/show.html.erb

```diff
@@ -88,7 +88,12 @@
       </ul>
     <% end %>
   <% end %>
-
+   <% if @view_object.assessment.recommendation_assessor_note.present?%>
+    <p class="govuk-body">
+     <%="#{@view_object.assessment.recommendation_assessor_note}"%>
```

Comment:
```suggestion
      <%= @view_object.assessment.recommendation_assessor_note %>
```

You don't need to interpolate it in to a string if it's the only thing being output.